subsys/profiling/Kconfig

```diff
@@ -0,0 +1,14 @@
+# cOPYRIGHT (C) 2023 kns gROUP llc (yadro)
```

Comment:
This and other similar ones

```suggestion
# Copyright (c) 2023 KNS Group LLC (YADRO)
```

Reply:
I don't know how I let this happen. Thanks for the comment. Corrected it.